tests/src/python/test_qgsserver_wms_getfeatureinfo.py

```diff
@@ -1,4 +1,14 @@
 # -*- coding: utf-8 -*-
+from qgis.core import QgsProject
```

Comment:
Can you move those imports back to where they were?

Reply:
Python style tests where failing, so I did the formatting with autopep8 like proposed in the travis message. Don't you think it's incorrect style, when I change it back?

Reply:
autopep8 takes a lot of configuration parameters (like line length and apparently placement of imports) which the style tests don't take into account.

The prepare-commit hook should run them properly, running autopep8 manually should never be required.